mappings/47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253.json

```diff
@@ -0,0 +1,45 @@
+{
+    "subject": "47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253",
+    "url": {},
```

Comment:
@siddigsalahaldin,

great! But you need to completely remove the field. Try that:
`{
    "subject": "47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253",
    "name": {
        "sequenceNumber": 0,
        "value": "Taras",
        "signatures": [
            {
                "signature": "9799ec85f4339bbabfec51cf4335011168fedb3596c0584f60a34d65af40b4092763aff61921371a33b4f22d1919c36e46b7d95a0ad6d473ebe6b69b857cd703",
                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
            }
        ]
    },
    "ticker": {
        "sequenceNumber": 0,
        "value": "TRS",
        "signatures": [
            {
                "signature": "a0d679e6583ed0dcc9e51d71cdb9c1b5e40bb2a4997d0f7d3b9e941b761e0acd7e1a8eedb776c62dd3f7160a506175a9649d932d306a6a94053effbb72182e03",
                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
            }
        ]
    },
    "decimals": {
        "sequenceNumber": 0,
        "value": 0,
        "signatures": [
            {
                "signature": "8ab6072ddb892ec9c948deb7de5f686633b818318d4a52638888fa22113021ae6d4fd2b50c7ef492e1098706cca0182b4476216b7171d6dedc4c58b759edc208",
                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
            }
        ]
    },
    "policy": "820182018282051a0629c62d8200581c39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f",
    "description": {
        "sequenceNumber": 0,
        "value": "The Taras native token's main goal is to help solve the problem of digital transfers so that they are easier than before",
        "signatures": [
            {
                "signature": "e572a0c412c4a9bee2b0e2e86c4b4615c53520ee41283d07cf818e966c179ded395b0e5d573bed312448e626dfe341fb31e347d9f3058a089c8a643aa7eea00b",
                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
            }
        ]
    }
}`